deploy/cephfs/helm/values.yaml

```diff
@@ -26,7 +26,7 @@ attacher:
 
   image:
     repository: quay.io/k8scsi/csi-attacher
-    tag: v1.1.1
+    tag: canary
```

Comment:
why we are using canary tagged attacher?

Reply:
@Madhu-1 interesting question :) . the previous tag is actually "2" months old and this 'canary' is ~1.15 release. So the option is either use 2 months old tag or use latest one. I am not sure there going to be a new version out soon. if yes, we can pull it whenever avaliable .

Reply:
instead of using the canary (which is kind of bleeding edge images) let's ask for Kube people to do a release or stick to the released version.

Reply:
few hours ago `v1.2.0` tag was cut and I replaced canary with that tag